### DISCUSSION TITLE
We are contributing 🎉🎉🎉Video-LLaVA

### DISCUSSION THREAD

**Discussion by LinB203** [haotian-liu/LLaVA#846] [Show and tell]
Hello, esteemed LLaVA developer, thank you for contributing such robust code and data to the community.

We have extended LLaVA to [Video-LLaVA](https://github.com/PKU-YuanGroup/Video-LLaVA) to achieve advanced performance on MSRVTT,MSVD,TGIF,ACTIVITYNET.

Thank you again for your contributions to the large visual-language model!

<img width="440" alt="sota" src="https://github.com/haotian-liu/LLaVA/assets/62638829/67c5dd4f-15e7-4b52-b156-a6af1ff78df8">

**Comment by ChunyuanLI** [comment]
Congrats to the great achievement. It is definitely an important direction to equip large multimodal models with video understanding capability.
